modules/apps/transfer/client/cli/query.go

```diff
@@ -167,3 +167,36 @@ func GetCmdQueryDenomHash() *cobra.Command {
 	flags.AddQueryFlagsToCmd(cmd)
 	return cmd
 }
+
+// GetCmdQueryTotalEscrowForDenom defines the command to query the total amount of tokens in escrow for a native denom
+func GetCmdQueryTotalEscrowForDenom() *cobra.Command {
```

Comment:
We will need to register the cmd on the cli, see [here](https://github.com/stackman27/ibc-go/blob/stack/total-ibc-out/modules/apps/transfer/client/cli/cli.go#L17)